src/components/sbb-form-field/sbb-form-field.tsx

```diff
@@ -139,8 +139,9 @@ export class SbbFormField implements ComponentInterface {
         (e): e is HTMLElement =>
           e.tagName === 'INPUT' ||
           e.tagName === 'SELECT' ||
-          e.classList.contains('sbb-form-field-element')
+          e.getAttribute('data-sbb-form-field') !== null
```

Comment:
would it be possible to use hasAttribute?
```suggestion
          e.hasAttribute('data-sbb-form-field')
```